### PR TITLE
console commands for list management

### PR DESCRIPTION
**Pull request by danschwarz**
List view, create, delete, view accounts for one list. Also, list_add_account, list_remove_account.

This PR incorporates PR #340 , which I will now close.

I'll add integration tests if I can get a local Mastodon test instance up & running on my system. Hopefully soon.

### CODE REVIEW

**Comment by danschwarz** [comment]
Added `toot list_add_account` and `toot list_remove_account` commands.  This makes the list-handling command set feature-complete.

**Review by ihabunek — CHANGES_REQUESTED** [review]
Hey thanks for this. Mostly looks good, I have some suggestions for change, mostly to make it less verbose.

File: toot/tui/scroll.py [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # scroll.py
 #
 # Copied from the stig project by rndusr@github
-# https://github.com/rndusr/stig
+# https://github.com/rndusr/sti
 #
```
[the remark this answers is not in the document]
Comment (danschwarz) [reply toot/tui/scroll.py]: Yes I always do a `git diff` sanity check; this must have gotten through when I wasn't paying close enough attention. I haven't tried `commit -p`; I'll do that.

File: toot/console.py [hunk 2]
```diff
@@ -724,6 +724,62 @@ def editor(value):
...
+        require_auth=True,
+    ),
+    Command(
+        name="list_create",
+        description="Create a list",
+        arguments=[
+            (["--id"], {
+                "type": str,
+                "help": "ID of the list"
+            }),
```
[the remark this answers is not in the document]
Comment (danschwarz) [reply toot/console.py:755]: fixed

File: toot/console.py [hunk 3]
```diff
@@ -724,6 +724,62 @@ def editor(value):
...
+            (["--title"], {
+                "type": str,
+                "help": "title of the list"
+            }),
```
[the remark this answers is not in the document]
Comment (danschwarz) [reply toot/console.py:759]: fixed

File: toot/console.py [hunk 4]
```diff
@@ -724,6 +724,100 @@ def editor(value):
...
+        description="Delete a list",
+        arguments=[
+            (["--id"], {
+                "type": str,
+                "help": "ID of the list"
+            }),
+            (["--title"], {
+                "type": str,
+                "help": "title of the list"
+            }),
```
[the remark this answers is not in the document]
Comment (danschwarz) [reply toot/console.py:778]: This works. I'll implement.

File: toot/console.py [hunk 5]
```diff
@@ -724,6 +724,100 @@ def editor(value):
...
+        ],
+        require_auth=True,
+    ),
+    Command(
+        name="list_add_account",
```
[the remark this answers is not in the document]
Comment (danschwarz) [reply toot/console.py:783]: Fixed

File: toot/console.py [hunk 6]
```diff
@@ -724,6 +724,100 @@ def editor(value):
...
+            }),
+            (["--account"], {
+                "type": str,
+                "help": "Account to add"
+            }),
+        ],
+        require_auth=True,
+    ),
+    Command(
+        name="list_remove_account",
```
[the remark this answers is not in the document]
Comment (danschwarz) [reply toot/console.py:802]: Fixed

File: toot/commands.py [hunk 7]
```diff
@@ -423,6 +424,60 @@ def tags_followed(app, user, args):
...
+        print_out("<red>List not found</red>")
+        return
+    account = find_account(app, user, args.account)
+    if not account:
+        print_out("<red>Account not found</red>")
+        return
+    try:
+        api.add_accounts_to_list(app, user, list_id, [account['id']])
+        print_out(f"<green>✓ Added account \"{args.account}\"</green>")
+    except Exception as ex:
```
[the remark this answers is not in the document]
Comment (danschwarz) [reply toot/commands.py]: removed

**Comment by ihabunek** [comment]
Thanks! I merged this manually and added some tests.

I changed the "toot lists" command to display as a table, which seemed nicer.

**Pull request closed without merging**